code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -112,3 +112,24 @@
 /obj/item/projectile/bullet/seed
 	damage = 4
 	stamina = 1
+
+//////////////////
+//    20 BORE   //
+//////////////////
+
+/obj/item/projectile/bullet/pellet/shotgun_buckshot/twentybore_buckshot
+	name = "20g buckshot pellet"
+	damage = 10
+
+/obj/item/projectile/bullet/shotgun_slug/twentybore_slug
+	name = "12g shotgun slug"
+	damage = 50
```

Comment:
As the above. 30% of 50 is still only 35. Comparing to 12g slugs (60 to 42), still not bad at all.more importantly I can wipe out their limbs with this still, where their armor isn't.

Reply:
I've looked at this like ten times and it has now only clicked that this is called "12g shotgun slug", I'm an idiot.
It's not that much damage for a shotgun, but it's the only way I can make the shotgun an alternative to the improvised rifle instead of a direct upgrade in every way. With the 12G slugs doing 60 brute, the same as the rifle bullet, the rifle is just kinda left in the cold because it can't be sawn off and its firing action is much slower than the shotgun's firing action.
Initially I thought I could just make the shotgun more expensive to craft, but the thing is, the only thing this achieves is that instead of taking three minutes to get a shotgun, it takes four minutes.
I think it might have been Justin who initially suggested different ammo types? I can't remember, heck.

(Thanks for pointing out I misnamed the slug though heck that's an embarrassing msitake.)